pkg/custominjections/custominjections.go

```diff
@@ -0,0 +1,112 @@
+package custominjections
+
+import (
+	"encoding/json"
+	"reflect"
+	"strings"
+	"sync"
+
+	"github.com/golang/glog"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/k8snetworkplumbingwg/network-resources-injector/pkg/types"
+)
+
+const (
+	customInjectionsMainKey = "customInjections"
+)
+
+// UserDefinedInjections user defined injections
+type UserDefinedInjections struct {
+	sync.Mutex
+	Patchs map[string]types.JsonPatchOperation
+}
+
+// CreateUserInjectionsStructure returns empty UserDefinedInjections structure
+func CreateUserInjectionsStructure() *UserDefinedInjections {
+	var userDefinedInjects = UserDefinedInjections{Patchs: make(map[string]types.JsonPatchOperation)}
+	return &userDefinedInjects
+}
+
+// SetCustomizedInjections sets additional injections to be applied in Pod spec
+func (userDefinedInjects *UserDefinedInjections) SetCustomizedInjections(injectionsCm *corev1.ConfigMap) {
```

Comment:
Please use the consistent interface name as in line 95.
Either change `userDefinedInjects` to `injections` in this line or change `injections` to `userDefinedInjects` in line 95.

Reply:
Done.